src/views/GovernanceReviewTeam/index.tsx

```diff
@@ -499,6 +504,16 @@ const GovernanceReviewTeam = () => {
       <Footer />
     </PageWrapper>
   );
+
+  if (isUserSet && !user.isGrtReviewer(userGroups)) {
+    return <NotFound />;
+  }
+
+  if (isUserSet && user.isGrtReviewer(userGroups)) {
```

Comment:
I would prefer to nest this inside the conditional above. I think it's easier to see an `else` clause than to catch the `!`.

```
if (isUserSet) {
  if (user.isGrtReviewer(userGroups)) {
    return <RenderPage />;
  } else {
    return <NotFound />;
  }
}